server/repository/src/migrations/v2_01_00/activity_log.rs

```diff
@@ -7,6 +7,7 @@ pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
             r#"
                 ALTER TYPE activity_log_type ADD VALUE 'ASSET_PROPERTY_CREATED';
                 ALTER TYPE activity_log_type ADD VALUE 'VACCINE_COURSE_CREATED';
+                ALTER TYPE activity_log_type ADD VALUE 'PROGRAM_CREATED';
```

Comment:
Are they used anywhere? can't find any references to the string values in the code, or how does it work?

Reply:
It's generated by the enum with SCREAMING_SNAKE_CASE...
Here:https://github.com/msupply-foundation/open-msupply/blob/34feb1426e4484af539f72c1adcabb73b58c151e/server/repository/src/db_diesel/activity_log_row.rs#L29
Would be nice if we could use enums without Postgres Needing this. I feel like we should be able to do a mapping to TEXT Field in diesel...

Reply:
Ah yes right, should have known...